dynawo/3rdParty/omcUpdate_1_9_4/configure-ac-omcompiler.patch

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+# Patch for OpenModelica 1.9.4 (https://www.openmodelica.org)
+#
+# All rights reserved.
+#
+# THIS PROGRAM IS PROVIDED UNDER THE TERMS OF GPL VERSION 3 LICENSE OR
+# THIS OSMC PUBLIC LICENSE (OSMC-PL) VERSION 1.2.
+# ANY USE, REPRODUCTION OR DISTRIBUTION OF THIS PROGRAM CONSTITUTES RECIPIENT'S ACCEPTANCE
+# OF THE OSMC PUBLIC LICENSE OR THE GPL VERSION 3, ACCORDING TO RECIPIENTS CHOICE.
+#
+# The OpenModelica software and the Open Source Modelica
+# Consortium (OSMC) Public License (OSMC-PL) are obtained
+# from OSMC, either from the above address,
+# from the URLs: http://www.ida.liu.se/projects/OpenModelica or
+# http://www.openmodelica.org, and in the OpenModelica distribution.
+# GNU version 3 is obtained from: http://www.gnu.org/copyleft/gpl.html.
+#
+# This program is distributed WITHOUT ANY WARRANTY; without
+# even the implied warranty of  MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE, EXCEPT AS EXPRESSLY SET FORTH
+# IN THE BY RECIPIENT SELECTED SUBSIDIARY LICENSE CONDITIONS OF OSMC-PL.
+#
+# See the full OSMC Public License conditions for more details.
+# SPDX-License-Identifier: GPL-3.0 OR LicenseRef-OSMC-PL
+Index: OMCompiler/configure.ac
+===================================================================
+--- OMCompiler/configure.ac
++++ OMCompiler/configure.ac
+@@ -155,7 +155,7 @@ AC_ARG_WITH(msys, [  --with-msys[=no]              Point to an msys installation
+     LDFLAGS="$LDFLAGS -L$withval/$MINGWNUM/lib"
+     echo "Got: $CPPFLAGS"
+   ],
+-  [
++  [:
```

Comment:
What is the need for this change?

Reply:
Building on old centos with old version of autoconf